code-challenges.js

```diff
@@ -12,27 +12,47 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+const divBy3 = (number) => {
+  if (number % 3 === 0) {
+    return `${number} is divisible by three`
+  } else {
+    return `${number} is not divisible by three`
+  }
+}
 
-
+console.log(divBy3(num1));
+console.log(divBy3(num2));
+console.log(divBy3(num3));
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const firstLetterCap = (array) => {
+  return array.map(value => value.charAt(0).toUpperCase() + value.slice(1))
+}
 
-
+console.log(firstLetterCap(randomNouns));
 
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const onlyNums = (array) => {
+  // first find/filter the numbers
+  let numArray = []
+  numArray = array.filter(value => (typeof value === "number"))
 
+  // then sort the numbers from least to greatest
+  // by sorting based on the difference between the 2 items being compared
+  return numArray.sort((a,b)=>a-b);
+}
```

Comment:
Excellent working code!!!

Nothing wrong at all with how you wrote this, its fantastic, but I do want to show you another way we can do that single line return like you used above in a previous challenge-

`const onlyNums = (array) => {`
 ` return array.filter(value => (typeof value === "number")).sort((a,b)=>a-b);`
`}`

What we see here is exactly the same thing you had, but all in one return and chaining on the methods in their order of operation. 

Just wanted to show you another way we can keep our use of extra variables and returns down.